src/07-projects/project-items.apib

```diff
@@ -134,3 +134,17 @@ Update a project item's internal cost.
         + internal_cost_per_unit (Money, required, nullable)
 
 + Response 204 (application/json;charset=utf-8)
+
+### projectItems.updateInternalHourlyCost [POST /projectItems.updateInternalHourlyCost]
```

Comment:
Q: Shouldn't this be internal hourly rate, instead of internal hourly cost?

Reply:
See my comment in the implementation PR: https://github.com/teamleadercrm/core/pull/9365#discussion_r276610350

Reply:
Agree in theory, lets discuss on slack ;)

Reply:
We didn't find a better name. Closing this remark :D